mscore/qml/palettes/Palette.qml

```diff
@@ -552,7 +552,7 @@ GridView {
                 // force not hiding palette cell if it is being dragged to a score
                 enabled: paletteCell.paletteDrag
                 target: mscore
-                onElementDraggedToScoreView: paletteCell.paletteDrag = false
+                function onElementDraggedToScoreView() { paletteCell.paletteDrag = false; }
```

Comment:
Can detect any negative impact of this and the other changes in this commit on the AppVeyor Build using Qt 5.9

Reply:
"Cannot"?

Reply:
Why is this needed? This is intended to be a handler to existing signal, not a standalone function. I am not sure whether it will work so if declared with `function` keyword.

Reply:
> "Cannot"?
right, corrected

> Why is this needed? This is intended to be a handler to existing signal, not a standalone function. I am not sure whether it will work so if declared with `function` keyword.
Well, as written, I'm not really sure what I'm doing here, but did exactly as that runtime warning told me to do, and it seems to work.

Reply:
What does that warning say?

Reply:
Hold on, rebuilding without that change to check...

Reply:
Warning: qrc:/qml/palettes/Palette.qml:551:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... } (qrc:/qml/palettes/Palette.qml:551, )